File: profile/angular/src/app/shared/paging/filtered-page-req-tpl.ts
import { FilteredPageSearchParam } from './filtered-page-search-param';
export class FilteredPageReqTpl<T> {
    pageNum: number = 0;
    pageSize: number = 10;
    sortBy: string = "";
    sortAsc: boolean = true;
    searchCols: string = "";
    searchParams: FilteredPageSearchParam = new FilteredPageSearchParam();
}

File: profile/maven2/demo/src/main/java/com/example/demo/dto/FilteredPageReqTpl.java
package com.example.demo.dto;

/**
* @class FilteredPageReqTpl
* 
*/

public class FilteredPageReqTpl<T> {
    public int pageNum;
    public int pageSize;
    public String sortBy;
    public boolean sortAsc;
    public String searchCols;
    public FilteredPageSearchParam searchParams;
}

